### PR TITLE
Mekhi/about

### PR DESCRIPTION
**Pull request by MekhiHart**
Created About Us page, added global css file

### CODE REVIEW

File: client/src/app/about/page.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    {
+      name: "Kate Steer",
+      role: "Backend",
+      description:
+        "Kate Steer is a software developer with experience in database management and architecture, well as application design. In her free time, she enjoys creating music and reading.",
+      linkedInURl: "https://www.linkedin.com/in/kate-steer-4149001a6/",
+      profilePic: "/about/profilepic_kate.png",
+    },
+    {
+      name: "Howie",
```
Comment (haophn01) [inline client/src/app/about/page.tsx:40]: Hey, my name is Hao Phan, you can call me "Howie" on daily basis, "Howie" is just a nickname :(((

**Review by haophn01 — CHANGES_REQUESTED** [review]
The rest looks good.

File: client/src/app/about/page.tsx (hunk 1 above)
Comment (MekhiHart) [reply client/src/app/about/page.tsx:40]: oops let me change that lmao

**Review by haophn01 — APPROVED** [review]
All good.

**Pull request merged**